portfolio/src/main/webapp/script.js

```diff
@@ -34,53 +54,55 @@ sections.forEach((sectionName) => {
 
 navbar.appendChild(navbarList);
 
-// First section that opens is About Me
-getAboutMe();
-
 // Fetches a greeting from the server and adds it to the DOM.
 function getGreeting() {
     fetch('/greeting').then(response => response.text()).then((greeting) => {
         document.getElementById('greeting-container').innerText = greeting;
-  });
+    });
 }
 
 // Functions called by clicking on the sections from the navbar
 function getAboutMe() {
-    $("#wrapper").load("about-me.html");
+    window.location.hash = 'about-me';
+    $('#wrapper').load('about-me.html');
 }
 
 function getProjects() {
-    $("#wrapper").load("projects.html");
+    window.location.hash = 'projects';
+    $('#wrapper').load('projects.html');
 }
 
 function getContact() {
-    $("#wrapper").load("contact-me.html");
+    window.location.hash = 'contact';
+    $('#wrapper').load('contact-me.html');
 }
 
 function getGallery() {
-    const wrapper = document.getElementById("wrapper");
+    window.location.hash = 'gallery';
+
+    const wrapper = document.getElementById('wrapper');
     wrapper.textContent = '';
 
-    const galleryTitle = document.createElement("h1");
-    galleryTitle.textContent = "Gallery";
+    const galleryTitle = document.createElement('h1');
+    galleryTitle.textContent = 'Gallery';
     wrapper.appendChild(galleryTitle);
 
-    const galleryText = document.createElement("p");
-    galleryText.textContent = "Photography sparks my interest - to me, taking a photo " + 
-        "means freezing a moment that would, otherwise, be impossible to reproduce. " +
-        "Here are some photos I took or were taken of me:";
+    const galleryText = document.createElement('p');
+    galleryText.textContent = 'Photography sparks my interest - to me, taking a photo ' + 
+        'means freezing a moment that would, otherwise, be impossible to reproduce. ' +
+        'Here are some photos I took or were taken of me:';
     wrapper.appendChild(galleryText);
 
-    const imageContainer = document.createElement("div");
-    imageContainer.setAttribute("class", "image-container")
+    const imageContainer = document.createElement('div');
+    imageContainer.setAttribute('class', 'image-container')
 
     const totalImagesNumber = 6;
     
     for (let imageIndex = 1; imageIndex <= totalImagesNumber; imageIndex++) {
-        const imgUrl = "images/image-" + imageIndex + ".jpg";
-        const imgElement = document.createElement("img");
+        const imgUrl = 'images/image-' + imageIndex + '.jpg';
```

Comment:
I'm not feeling strongly about this, but I wanted to point out that you could use `images/image-${ imageIndex}.jpg` to compose strings as well.

Reply:
Indeed! I changed my code because I think it looks cleaner with `...${...}...`